.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+## Component
```

Comment:
Do we need this? Maybe labels with component names are more suitable?

Reply:
Removed